firefox-ios/Client/Frontend/Components/ContentContainer.swift

```diff
@@ -31,6 +31,10 @@ class ContentContainer: UIView {
         return type == .privateHomepage
     }
 
+    var hasWebView: Bool {
```

Comment:
nit; I think `var type: ContentType?` can be made private? I don't remember why I had set it as internal 😆

Reply:
yes, good call out! it doesn't seem to be used anywhere and we'll want to use the computed vars to check the type regardless.

Reply:
updated!